### PR TITLE
Install core plugins on first run

### PR DESCRIPTION
**Pull request by dustinmm80**

### CODE REVIEW

File: lib/conjur/cli.rb [hunk 1]
```diff
@@ -69,6 +71,17 @@ def run args
         super args
       end
 
+      def install_core_plugins
+        to_install = CORE_PLUGINS - Conjur::Config.plugins
+        unless to_install.empty? or ENV['CONJUR_SKIP_PLUGINS']
+          puts 'Installing core plugins'
+          to_install.each do |plugin|
+            puts "- #{plugin}"
+            self.run ['plugin', 'install', plugin]
```
Comment (kgilpin) [inline lib/conjur/cli.rb]: This is going to fail in customer environments where access to Rubygems is prohibited. Such as Box.

**Comment by kgilpin** [comment]
What's the rationale behind this?

**Comment by dustinmm80** [comment]
I'm trying to find a way to install plugins by default - omnibus is being a pain so i'm running this experiment on the side. Can you think of another approach?

**Comment by kgilpin** [comment]
Well, I guess we could just make it not-a-plugin. Because even if we install the gem into Omnibus, it still won't be loaded if it's not in .conjurrc.

**Comment by dustinmm80** [comment]
Yeah I think I have a workaround for that, trying it now. This PR will be deleted once I get it working

**Pull request closed without merging**